### DISCUSSION TITLE
saving more info in filters state

### DISCUSSION THREAD

**Discussion by emehran** [TanStack/table#2025] [General] [answered]
Hey
I was looking for a way to make a filter lists component. Currently, the state that returns from userFilters has only "id", and "value".
My question is is there any way to save more information about that filter in state? like its label (Header)?
Thanks

**Comment by gargroh** [comment, marked as answer]
You can make use of `value` key itself, e.g 

`value: { header: 'abc', 'txt': 'xyz', list: ['a', 'b', 'c'] }`
